server/aws-lsp-codewhisperer/src.gen/@amzn/codewhisperer-streaming/package.json

```diff
@@ -63,7 +63,7 @@
         "concurrently": "7.0.0",
         "rimraf": "^3.0.0",
         "typedoc": "0.23.23",
-        "typescript": "~5.4.5"
+        "typescript": "~4.9.5"
```

Comment:
Typedoc is usually not a mandatory dependency, is it not possible to upgrade the typedoc version to use the one dependabot suggests?